src/gtfs/read.rs

```diff
@@ -35,6 +35,17 @@ fn default_agency_id() -> String {
     "default_agency_id".to_string()
 }
 
+fn get_agency_id(route: &Route, networks: &CollectionWithId<objects::Network>) -> Result<String> {
+    route
+        .agency_id
+        .as_ref()
+        .map(|id| Ok(id.to_string()))
+        .unwrap_or_else(|| match networks.values().next() {
+            Some(n) => Ok(n.id.clone()),
```

Comment:
I don't like that: it means "if there is no agency_id, use the first one", but I think it should be "if there is no agency_id and we have only one agency, use the first one".

@prhod ?

Reply:
Agreed with @TeXitoi about the extact rule